css-view-transitions-1/Overview.bs

```diff
@@ -887,13 +886,17 @@ urlPrefix: https://wicg.github.io/navigation-api/; type: interface;
 
 			Note: This transform is applied to the snapshot using the `transform` property of the associated ''::view-transition-group'' pseudo-element.
 
+		* Effects on the element, such as 'opacity' and 'filter' are applied to the capture.
+
 		* [=list/For each=] |descendant| of [=shadow-including descendant=] {{Element}} and [=pseudo-element=] of |element|,
 			if |descendant| has a [=computed value=] of 'view-transition-name' that is not ''view-transition-name/none'',
 			then skip painting |descendant|.
 
 			Note: This is necessary since the descendant will generate its own snapshot which will be displayed and animated independently.
 
 			Issue: Refactor this so the algorithm takes a set of elements that will be captured. This centralizes the logic for deciding if an element should be included or not.
+
+			Issue: Specify what happens with 'mix-blend-mode'.
```

Comment:
It's ignored. This property is only relevant when blending an element into its target and doesn't need to impact the snapshot.

How this snapshot is blended into its target is determined by the pseudo-element hosting it.

Reply:
Why non-normative rather than normative?

Reply:
I figured it's implicit because mix-blend-mode is only relevant when an element is drawn into its target. The UA styles we add on the pseudo-element is because the corresponding DOM element's mix-blend-mode might not be accurate for what's needed during the transition. So what else could we do with the DOM element's mix-blend-mode.

I'm fine with normative text for this too.

Reply:
Would it be more correct to say the element is rendered within a new stacking context? That effectively ignores `mix-blend-mode` on the root, but explains what happens for `mix-blend-mode` on inner elements.